### DISCUSSION TITLE
Homepage (for you) section is broken

### DISCUSSION THREAD

**Discussion by remahmoud** [community/community#55493] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

![GitHub](https://github.com/community/community/assets/130402111/13557582-230e-41f6-a980-2ec79b2a5cde)

**Comment by samanyuneelson** [comment]
same for me (only on smaller screens)
![GitHub - Brave 5_15_2023 8_01_20 PM](https://github.com/community/community/assets/22994948/2b722502-5817-4301-87fb-4baa87bc80ff)

**Reply by XxSonicGhostxX** [reply]
wtf

**Comment by torn4dom4n** [comment]
It happens on both my laptop and my phone.

**Reply by remahmoud** [reply]
they fix the bug now 👍

**Comment by Nukealbert** [comment]
Bug is still coming to my home screen

**Reply by remahmoud** [reply]
@Nukealbert  i think it's browser cache you can refresh the cache by `Ctrl+F5`

**Reply by saxbophone** [reply]
> @Nukealbert i think it's browser cache you can refresh the cache by `Ctrl+F5`

I'm doing <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>R</kbd> for force-refresh with Chrome on Mac and the layout issue is still persisting, are you sure it's been fixed?

**Reply by remahmoud** [reply]
@saxbophone  I think not all the servers have been updated, because the problem has been fixed with me.

![GitHub](https://github.com/community/community/assets/130402111/87bbdaeb-3654-4dc6-b508-a686e6e7b1fb)

**Reply by saxbophone** [reply]
> @saxbophone I think not all the servers have been updated, because the problem has been fixed with me.
> 
> ![GitHub](https://user-images.githubusercontent.com/130402111/238458695-87bbdaeb-3654-4dc6-b508-a686e6e7b1fb.png)

Ah, gotcha, thanks! I even tried restarting my browser and still got the glitch, so I think you're probably right, we just have to wait for the change to propagate.

**Comment by saxbophone** [comment]
Can confirm that I get the same issue:
![image](https://github.com/community/community/assets/8693463/739d0990-ee97-4180-9628-087c637623e9)

**Comment by Kreuger** [comment]
I thought this was a Firefox issue. Clearing the cache didnt help me.

**Comment by Newbie012** [comment]
![Screenshot_20230516-095541](https://github.com/community/community/assets/10504365/cc6a328c-d746-41fe-a681-520cb4b36785)

Broken on mobile

**Comment by ghost** [comment]
Also broken here clear cache don't fix (maybe a propagation of the fix problem).

**Comment by davidf9265** [comment]
Same for me, any size of screen. Issue seems to be related with my specific content (compared with my friend feed and bug didn't appear on his account). Seems to be easy to fix, container styles appear not to be very responsive or relative.

**Comment by tarikjabiri** [comment]
Same here

**Comment by remahmoud** [comment]
The bug has not been fixed yet :confused:

![GitHub](https://github.com/community/community/assets/130402111/df59b0c6-ffee-4253-8dc1-336955baa1ad)

**Comment by lodembeep** [comment]
Don’t tell me that people at microsoft are unable to fix a simple frontend issue …

the bug is still live on mobile (ios)

**Comment by saxbophone** [comment]
I can confirm that the issue still persists for me. After 3 days, I don't think it's a cache propagation issue. May be more likely that manifestation of the bug depends on what content is displayed on the page, hence why some users see it and others don't:
![image](https://github.com/community/community/assets/8693463/a390c77b-aac1-4d15-97e6-cd1aa0ff8886)

**Reply by ghost** [reply]
> I can confirm that the issue still persists for me. After 3 days, I don't think it's a cache propagation issue. May be more likely that manifestation of the bug depends on what content is displayed on the page, hence why some users see it and others don't: ![image](https://user-images.githubusercontent.com/8693463/239092556-a390c77b-aac1-4d15-97e6-cd1aa0ff8886.png)

i don't think so i had the bug and it fixed itself an hour after with nothing else changed (same content etc) maybe a CDN propagation issue or an ISP cache.

**Reply by lodembeep** [reply]
An ISP has nothing to do with that.
This is a clear CSS issue.

most likely a propagation issue and/or dev issue.

if you want to test for the bug, just try it on mobile. And yes, it is always the same element screwing the styles.

i should add that, i don’t have any issue on my laptop on the highest resolution. Only on mobile.

**Reply by remahmoud** [reply]
@saxbophone The problem appears to me sometimes and sometimes it doesn't, I don't know whether they fixed it or not, but it is a matter of time.

**Reply by ghost** [reply]
> An ISP has nothing to do with that. This is a clear CSS issue.
> 
> most likely a propagation issue and/or dev issue.
> 
> if you want to test for the bug, just try it on mobile. And yes, it is always the same element screwing the styles.
> 
> i should add that, i don’t have any issue on my laptop on the highest resolution. Only on mobile.

It seem you don't know that most major isp do have CDN in their datacenter to cache most used thing like ... CSS or picture.....

**Reply by lodembeep** [reply]
After 48 hours, it is neither ISP nor CDN or cache

**Reply by ghost** [reply]
> After 48 hours, it is neither ISP nor CDN or cache

IT seem you clearly don't know how cache of this size do work and how many time it can remain in the worse case ...

**Reply by lodembeep** [reply]
IT doesn’t matter since they must be standard …

**Reply by ghost** [reply]
> IT doesn’t matter since they must be standard …

OH god yet another i know everything that now nothing and talk about standard i'm out

**Reply by saxbophone** [reply]
Please stop arguing about which of you is more knowledgeable about ISPs, CDNs and caching. Most people are reading here to focus on the issue at hand.

**Reply by lodembeep** [reply]
> > IT doesn’t matter since they must be standard …
> 
> OH god yet another i know everything that now nothing and talk about standard i'm out

…

**Comment by saxbophone** [comment]
@davidjarzebowski Any chance we can get an update from Github about whether or not youse are aware of this UI issue?

**Reply by ghost** [reply]
The ui is fixed on the "main" (and replicated to some in some EU server (since my vpn there also don't have the issue anymore (since this morning) so i guess it will come very soon everywhere)

**Reply by lodembeep** [reply]
I am on EU and i still have the issue

**Reply by saxbophone** [reply]
Do you have a source for that? It was claimed earlier that it had been updated and yet it's been three days with no change for some of us. Seems an awfully long propagation time, no?

**Reply by ghost** [reply]
> I am on EU and i still have the isssue

can you learn to read ? "in SOME eu"

**Reply by ghost** [reply]
> Do you have a source for that? It was claimed earlier that it had been updated and yet it's been three days with no change for some of us. Seems an awfully long propagation time, no?

yes my own use on my vpn in hertzner in germany (but i cannot talk about all server in all EU)

unlike some of us here @lodembeep i don't take my case (germany vpn) to a generality and talk nonsense about every server in eu.

I think it will be a valid "new" bug if tomorrow you don't see the fix ... i guess ?

**Reply by saxbophone** [reply]
> > Do you have a source for that? It was claimed earlier that it had been updated and yet it's been three days with no change for some of us. Seems an awfully long propagation time, no?
> 
> yes my own use on my vpn in hertzner in germany (but i cannot talk about all server in all EU)

I meant, do you have a source with positive confirmation from Github that they've had an issue and rolled out a fix (I mean like a bugfix announcement or something, not first-hand experience, though your experience is noted).

**Reply by lodembeep** [reply]
![E96F3E3B-6F2B-4754-A253-1E9E5227CA11](https://github.com/community/community/assets/48884244/cf1af857-9207-4031-9176-dec12368c354)


Still not fixed/propagated

**Reply by ghost** [reply]
> > > Do you have a source for that? It was claimed earlier that it had been updated and yet it's been three days with no change for some of us. Seems an awfully long propagation time, no?
> > 
> > 
> > yes my own use on my vpn in hertzner in germany (but i cannot talk about all server in all EU)
> 
> I meant, do you have a source with positive confirmation from Github that they've had an issue and rolled out a fix (I mean like a bugfix announcement or something, not first-hand experience, though your experience is noted).

ah that no, but since many user had the problem and it's fixed (and i do not think it's fixed itself), i guess someone have fixed it, (or unbroken it (depending of what's the bug is))

but i do think they will do a post mortem once fixed (if it's a bug comming from them ?)

**Reply by ghost** [reply]
> ![E96F3E3B-6F2B-4754-A253-1E9E5227CA11](https://user-images.githubusercontent.com/48884244/239095932-cf1af857-9207-4031-9176-dec12368c354.jpeg)
> 
> Still not fixed/propagated

not because it's not fixed for you that it's automatically means that all EU is broken ya know ?

**Reply by saxbophone** [reply]
Thanks for confirming that —the reason I ask is because it's clear that some people are still experiencing the bug, others not, and I wanted to know if the claim that it had been fixed, was based on an assumption based on experience, or from some press release of some kind from GH. Anyway, it's still broken for some of us and not for others! 😅  Have a good night all, hopefully David can enlighten us about what the ding-dang-diddly-doo-glum-fiddle-dosh is going on! ☺️

**Reply by lodembeep** [reply]
GN.
Was fun to argue with you @BirdInFire

**Reply by queenofcorgis** [reply]
@saxbophone thanks for posting, I totally understand that this is frustrating, but please refrain from tagging individual GitHub staff members for updates as they may not be the best point of contact and it goes against the spirit of our [Code of Conduct ](https://github.com/community/community/blob/839d7bf3917451aea4f38ee31ce8c728767b9a29/CODE_OF_CONDUCT.md).

**Comment by TheMonDon** [comment]
I am also having this issue, thought I was the only one.

**Reply by ghost** [reply]
No we (nearly) all had it, but some of us already have received (we guess) a fix (since it work again), so normally (if it's really fixed) must come to you too soon.

**Comment by Caballero-Arepa** [comment]
![imagen](https://github.com/community/community/assets/78449553/77a42c1e-5c13-4f09-bb1f-6a702437a77d)
Same here

**Comment by queenofcorgis** [comment]
Hi there from GitHub 👋🏼 we are tracking this internally and investigating further. We appreciate your patience in the interim. I'll share an update here when I have one. Thank you all for your reporting, screenshots, and additional input.

**Comment by ArjixWasTaken** [comment]
## I was about to make a new discussion for this, but I found that one already exists, so I'll post it here instead.

This is how the "For you" feed should look like:
![image](https://github.com/community/community/assets/53124886/10dc9602-57ae-4f5f-8ef6-6d97b7db6c6a)

And this is how it looks like after I scroll to the bottom, and click the "More" button to load more items from the feed.
![image](https://github.com/community/community/assets/53124886/bcf0545a-9b21-416b-ac15-d59e633b8149)

I don't know what's causing it, I tried on a browser w/o custom CSS and the same thing happened there.

PS: The "Following" feed behaves as expected, this only happens with the "For you" feed.

**Reply by ArjixWasTaken** [reply]
PS2: Yes I know that it is redundant to give a before and after screenshot, as the title says, I originally planned to make a separate issue for this as I wasn't aware of an existing one.

**Comment by XxSonicGhostxX** [comment]
i need help im trying to setup server so maybe i can stop with the bs and get my iphone accounts unlocked and he able to see everything as it should be. i need some guidance useing secure shell to connect to server what options do i put it in for everything. also why when i make a key it says it didn’t work go to github. my other computer haw a bunch of apis on but noway to get internet. help me

**Reply by saxbophone** [reply]
@XxSonicGhostxX I think you replied to the wrong email thread bud

**Comment by XxSonicGhostxX** [comment]
what exactly is this code of conduct i may need that..

**Reply by remahmoud** [reply]
@XxSonicGhostxX you at the wrong place 👎

**Comment by XxSonicGhostxX** [comment]
if you say so ok.. well ido how this is in my email thread really needa figure this out, could you possibly still try to help me understand some process you DO kno that much right ?

**Reply by ArjixWasTaken** [reply]
First of all, don't reply from emails on discussions, it makes a new thread every time.

Second of all, we have no context on what you want to do, and your initial message amounts to no useful information to help you.

Either talk in the correct discussion/issue or just don't talk.

**Comment by XxSonicGhostxX** [comment]
Well then I apologize this is the last time I’ll reply from my email but I don’t have access to this account yet, as I was trying to just get some HELP . See had no clue about how I should respond ether so don’t,..that’s exactly why I reached out.. Answers to things I don’t know. Don’t worry about it guys I’ll figure it out myself. Thx real welcoming community for someone that’s just trying to find a good community here.

**Reply by saxbophone** [reply]
To be clear, people are criticising your posts because you seem to be talking about something completely unrelated to this Github discussion thread, about which you are getting emails about updates to because you commented on it some days ago. If you don't want to participate in this discussion or receive updates, you can click the unsubscribe link in the email notifications you get.

For reference, this thread is located at: https://github.com/orgs/community/discussions/55493

Hopefully, if you have a look at what we can see on our end, you can appreciate why your posts come across as spam as you seem to be asking questions about something completely unrelated to what this discussion thread is about (it's about a UI bug in Github).

If you've got general issues with your Github account, Github Support are the ones to contact, not this place (this is just a discussion thread among ordinary users).

**Reply by ArjixWasTaken** [reply]
> Well then I apologize this is the last time I’ll reply from my email but I don’t have access to this account yet

Github support is where you should be at if you can't access your account.

https://support.github.com/request

It was a simple google search, but no, you had to complain here.
If you can't even bother searching on google, how do you expect us to welcome you?

Why do you think search engines exist?
They exist to help us find what we are looking for.

**Comment by SunderB** [comment]
I can also confirm this has occurred to me on multiple devices (Safari on iPad & Firefox for Android). On phones it is basically unusable.
Here's a screenshot on the iPad
![96DCAC6F-280A-4BA7-9779-E2540DE4D161](https://github.com/community/community/assets/20426079/6bd215c8-6dca-4eef-9338-ab50eba01cc9)


'Following' does not have this issue.

**Reply by pachico** [reply]
Same for Brave. 
Is this a way to force people to install the mobile app?
Because that's what I had to do to have any readability.

**Comment by alflanagan** [comment]
Seeing this on 5/23/2023 on a fresh load of the page in Safari, on a Mac M1 notebook.

<img width="779" alt="Screenshot 2023-05-23 at 6 22 03 PM" src="https://github.com/community/community/assets/1546080/d06feb27-5a7d-48fd-b5e6-2edac84a9e91">

**Comment by saxbophone** [comment]
Don't look now, but it looks like it's fixed for me! :)

**Comment by queenofcorgis** [comment, marked as answer]
Confirming that this has been investigated and resolved. Please do let us know if you're continuing to encounter any issues.

**Reply by remahmoud** [reply]
Thanks 👍

**Reply by queenofcorgis** [reply]
Thank you @remahmoud and everyone else for reporting 🙇🏼‍♀️ !
